README.md

```diff
@@ -15,15 +21,64 @@ $ pip install .
 ```
 
 3) [Optional] Enable bash completion
+for green-cli:
 ```
 $ eval "$(_GREEN_CLI_COMPLETE=source green-cli)"
+```
+
+for green-liquid-cli:
+```
 $ eval "$(_GREEN_LIQUID_CLI_COMPLETE=source green-liquid-cli)"
 ```
 
-Example usage (testnet):
+# Hardware wallet support (via hwi) [BETA]
+
+To enable hardware wallet support (via the `--auth hardware` option)
+additional dependencies must be installed from requirements-hwi.txt.
+
+1) Install libudev and libusb. This is platform specific but for debian:
+```
+$ sudo apt-get install libudev-dev libusb-1.0-0-dev
+```
+
+2) Install extra requirements
+```
+$ pip install -r requirements-hwi.txt
+```
+
+You can now run green-cli (or green-liquid-cli) passing the `--auth
+hardware` option.
+
+# Software authenticator support (via libwally) [BETA]
+
+There is another authenticator option `--auth wally` which delegates the
+possession of key material (the mnemonic) and authentication services to
+python code external to the gdk using the hardware wallet interface.
+This is useful for testing and as a demonstration of the technique. In
+order to enable this option wally must be installed:
+
+```
+$ pip install -r requirements-wally.txt
+```
+
+# Example usage
+
+Recover a testnet wallet and report the balance:
```

Comment:
Maybe 'log in to' rather than recover (which suggests recovery of funds)

Reply:
done, thanks